keel-core/keel-core.gradle.kts

```diff
@@ -31,9 +31,10 @@ dependencies {
   api("com.netflix.frigga:frigga")
   api("com.netflix.spinnaker.kork:kork-core")
   api("net.swiftzer.semver:semver:1.1.0")
-  api("org.jetbrains.kotlinx:kotlinx-coroutines-core")
+  api("org.jetbrains.kotlinx:kotlinx-coroutines-jdk8")
```

Comment:
should this be jdk11?

Reply:
No, this library is a bit awkwardly named. It's a collection of extension functions for things that are only in JDK 8 +. In particular things that translate between Java's `Executor` and `CompleteableFuture` and Kotlin coroutines. It's usable on any Java version from 8 onward.

There's also a `kotlinx-coroutines-jdk9` (not sure exactly what's in that one), but not any specific to higher versions.

Reply:
hahaha wonderful. sounds fine